apps/console/src/features/applications/components/settings/sign-on-methods/script-based-flow/script-based-flow.tsx

```diff
@@ -33,21 +33,26 @@ import {
     Tooltip,
     useDocumentation
 } from "@wso2is/react-components";
+import { AxiosResponse } from "axios";
 import beautify from "js-beautify";
 import cloneDeep from "lodash-es/cloneDeep";
 import get from "lodash-es/get";
 import isEmpty from "lodash-es/isEmpty";
 import kebabCase from "lodash-es/kebabCase";
 import set from "lodash-es/set";
-import React, { FunctionComponent, ReactElement, useEffect, useRef, useState } from "react";
+import React, { ChangeEvent, FunctionComponent, ReactElement, useEffect, useRef, useState } from "react";
 import { Trans, useTranslation } from "react-i18next";
 import Joyride, { CallBackProps, STATUS, Step } from "react-joyride";
 import { useDispatch } from "react-redux";
-import { Checkbox, Icon, Menu, Sidebar } from "semantic-ui-react";
+import { Checkbox, Dropdown, Header, Icon, Input, Menu, Sidebar } from "semantic-ui-react";
 import { stripSlashes } from "slashes";
 import { ScriptTemplatesSidePanel } from "./script-templates-side-panel";
 import { getOperationIcons } from "../../../../../core/configs";
 import { AppUtils, EventPublisher } from "../../../../../core/utils";
+import { getSecretList } from "../../../../../secrets/api/secret";
```

Comment:
Lets have indexes in the subfolders of the feature so you can shorten these imports.
```suggestion
import { getSecretList } from "../../../../../secrets/api";
```

```
---api
   |---index
---utils
   |---index
```

Reply:
Will be fixed with another code change.